app/common/containers/pages/InnmsListPage/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { translate } from 'react-i18next';
+import { provideHooks } from 'redial';
+import withStyles from 'withStyles';
+import Helmet from 'react-helmet';
+
+import filter from 'helpers/filter';
+
+import { H1, H2 } from 'components/Title';
+import Pagination from 'components/Pagination';
+import Button from 'components/Button';
+
+import Table from 'components/Table';
+import ShowBy from 'containers/blocks/ShowBy';
+
+import SearchForm from 'containers/forms/SearchForm';
+
+import { getInnms } from 'reducers';
+
+import { fetchInnms } from './redux';
+import styles from './styles.scss';
+
+const FILTER_PARAMS = ['id', 'sctid', 'name', 'name_original'];
+
+@withRouter
+@withStyles(styles)
+@translate()
+@provideHooks({
+  fetch: ({ dispatch, location: { query } }) =>
+    dispatch(fetchInnms({ page_size: 5, ...query })),
+})
+@connect(state => ({
+  ...state.pages.InnmsListPage,
+  innms: getInnms(state, state.pages.InnmsListPage.innms),
+}))
+export default class InnmsListPage extends React.Component {
+  get activeFilter() {
+    const index = FILTER_PARAMS.indexOf(Object.keys(this.props.location.query)
+      .filter(key => ~FILTER_PARAMS.indexOf(key))[0]);
+    return FILTER_PARAMS[index !== -1 ? index : 0];
+  }
```

Comment:
We shouldn't iterate twice over `FILTER_PARAMS`. Also, its hard to read code with static array accessors and hacky array inclusion check constructions, try to avoid usage of them.